README.md

```diff
@@ -30,20 +30,46 @@ xcode-select --install
 
 # Available Actions
 ## iOS
-### ios release_appstore
+### ios add_single_device
 ```
-fastlane ios release_appstore
+fastlane ios add_single_device
 ```
-New release to `TestFlight` for Appstore.
+Adds a new device and regenerates the `Provisioning Profile`s to include it.
+### ios release_qa
+```
+fastlane ios release_qa
+```
+Run this before doing anything else
+
+Perform project configurations validations.
+
+After all the steps have completed succesfully, run this.
+
+Remove all build artifacts created by fastlane to upload.
+
+If there was an error, run this.
```

Comment:
These automatic comments generated with `desc` could be put to better use describing what the steps will do so this ends up being a nice summary of the lane's actions and doings.

Reply:
Done